### DISCUSSION TITLE
Propagator from `QobjEvo` for open systems

### DISCUSSION THREAD

**Discussion by ottosalmenkivi** [qutip/qutip-qip#174] [Q&A]
Hi qutip-qip comunity!

I facing a problem with generating a propagator for a incoherent processor. I have tried to demonstrate the problem in the attached notebook. All help is appreciated.

Thanks!

Otto Salmenkivi
[Propagator_with_SCQubits.zip](https://github.com/qutip/qutip-qip/files/9838507/Propagator_with_SCQubits.zip)

**Comment by hodgestar** [comment]
@ottosalmenkivi This seems to be more a question about QuTiP than about qutip-qip. You're welcome to ask on the qutip mailing list if your question has not been resolved yet.
